### DISCUSSION TITLE
screen for modification of online map source

### DISCUSSION THREAD

**Discussion by mariush444** [osmandapp/OsmAnd#13540] [Q&A] [answered]
As I understand because of new Android's rules it will be not possible to modify .metainfo file or modify the sqlite file by user.

Currently it is possible to add new source of online map and delete in "download map" screen
but there is not possible to modify the definition e.g. link, zoom or age (refresh time)

It seems that it is `must be` in near future.

**Comment by vshcherb** [comment, marked as answer]
- You can always do it via Export / Import.
- You can create a new tile source with UI in Android but it doesn't display all properties
- You still have access to files if you store it in Android Multiuser storage
